pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a placeholder:
+
+`gum input --placeholder "{{placeholder_string}}"`
+
+- Interactvely pick an option which will exit if "No" is chosen:
+
+`gum confirm "{{Continue?}}" --default=false --affirmative "{{Yes}}" --negative "{{No}}" {{&& : || exit 1}}`
```

Comment:
```suggestion
`gum confirm "{{Continue?}}" --default=false --affirmative "{{Yes}}" --negative "{{No}}" {{|| exit 1}}`
```

Reply:
IMO, yes since it gives the user an idea that you can add a command for if `Yes` is chosen.

Maybe we could change it to `{{&& echo "Yes selected" || echo "No selected"}}

Reply:
> aybe we could change it to {{&& echo "Yes selected" || echo "No selected"}}

I like this variant.